prime-angular-frontend/src/app/shared/components/address-autocomplete/address-autocomplete.component.html

```diff
@@ -8,14 +8,12 @@
         <input matInput
                placeholder="Address Autocomplete"
                formControlName="autocomplete"
-               required="false"
```

Comment:
@AnaisHebert controllers should be used to drive reactive forms, and this should be used only if not attached to a form in the controller